### PR TITLE
Updates to OIDC backends default config

### PR DESCRIPTION
**Pull request by VJalili**

### CODE REVIEW

File: config/oidc_backends_config.xml.sample [hunk 1]
```diff
@@ -14,6 +14,11 @@
 
          If you want the consent screen to be shown to the new users only, and re-authorization happen without
          asking for user's consent, then remove this attribute.
+
+         NOTE: Galaxy sets OIDC 'scope' (requested scope of access to user's account) to `openid`. This is the
+         minimum scope value that request only user's email address and profile name. For login process Galaxy
+         does not need any more information, hence we request minimum possible information. By design, Google
+         does NOT show consent screen for this scope, hence user only see's a login page.
```
Comment (martenson) [inline config/oidc_backends_config.xml.sample]: s/see's/sees

Comment (VJalili) [reply config/oidc_backends_config.xml.sample]: @martenson I did update the comment.

**Comment by martenson** [comment]
@VJalili If you want this in the release you need to target 18.09. Or is this fine in dev?

**Comment by VJalili** [comment]
I think this can safely target dev.

**Comment by VJalili** [comment]
@martenson Thank you.

**Pull request merged**